static/src/javascripts/projects/common/modules/analytics/forceSendMetrics.ts

```diff
@@ -0,0 +1,5 @@
+export let forceSendMetrics = false;
+
+export const setForceSendMetrics = (val: boolean): void => {
+	forceSendMetrics = val;
```

Comment:
I think that we need to call `coreVitals()` again, because it was probably initialised before this code ran.

If `val` is `true` different to `forceSendMetrics`, you know that it needs to be called again.